optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/query/event/process/FlowNodeInstanceDto.java

```diff
@@ -115,7 +115,7 @@ public FlowNodeInstanceDto(
     this.engine = engine;
     this.flowNodeId = flowNodeId;
     this.flowNodeInstanceId = flowNodeInstanceId;
-    this.flowNodeType = FLOW_NODE_TYPE_USER_TASK;
+    flowNodeType = FLOW_NODE_TYPE_USER_TASK;
```

Comment:
the removal of this only for this line is a little odd, shall we make it consistent in this file at least?